ctapipe/io/hdf5eventsource.py

```diff
@@ -613,6 +624,11 @@ def _generator(self):
                     for algorithm, readers in algorithms.items():
                         c[algorithm] = next(readers[key])
 
+                        # change prefix to new data model
```

Comment:
how is this `to new data model`?
It is only for the older version, right?

Reply:
If we have the old data model, we change the prefix to fit with the new version (so we don't lie when writing events out again).